src/main/resources/htmlpublisher/HtmlPublisherTarget/HTMLAction/sidepanel.jelly

```diff
@@ -1,8 +1,6 @@
 <!--
   This view is used to render the plugin side panel.
 
-  We don't really have anything here, This is only a placeholder. 
+  We don't really have anything here, This is only a placeholder.
 -->
-<div>
```

Comment:
When browsing the archived files directly, this `div` showed up repeatedly as it's not wrapped in a layout element.